corehq/apps/case_search/views.py

```diff
@@ -74,10 +74,14 @@ def post(self, request, *args, **kwargs):
 
         if xpath:
             search = search.xpath_query(self.domain, xpath)
+
+        search = search.enable_profiling()
```

Comment:
do you know if this drastically affects performance when there are a lot of cases? Might it be worth only doing this if the user requests?

Reply:
Good point, docs do say this adds substantial overhead. Split the two actions in ff43bd9cf0ae1e6fe255553c3a057b3ac2b6cf6e, giving them two separate buttons:

![Screen Shot 2021-01-11 at 12 50 35 PM](https://user-images.githubusercontent.com/1486591/104219231-db90b780-540b-11eb-92fa-63e3872e4552.png)